### PR TITLE
Implement build system based on redo

### PR DESCRIPTION
**Pull request by Ikke**
Builds the docs and allows you to install it using a build system based on [redo][].

[redo]:https://github.com/apenwarr/redo

### CODE REVIEW

File: install.do [hunk 1]
```diff
@@ -0,0 +1,28 @@
+. ./conf
+
+: ${DESTDIR:=NONE}
+: ${PREFIX:=/usr}
+: ${MANDIR:=$DESTDIR$PREFIX/share/man}
+: ${BINDIR:=$DESTDIR$PREFIX/bin}
+
+if [ "$DESTDIR" = "NONE" ]; then
```
Comment (maxice8) [inline install.do]: DESTDIR=/ will work for installing to the system like Makefile installations without DESTDIR set, correct ?

Comment (Ikke) [reply install.do]: Yes, it just forces you to specify something.

Comment (maxice8) [reply install.do]: good

**Comment by maxice8** [comment]
Merged, thank you.

**Pull request merged**